### ISSUE TITLE
Layout of games is broken

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/make-a-ten#309]
See screenshot below from main today.  All levels that I checked look this way.

Assigning to responsible dev @marlitas.

<img width="1067" alt="screenshot_3207" src="https://github.com/phetsims/make-a-ten/assets/3046552/c41c1e27-3473-44c3-a2c0-507a39f602b8">

**Comment by marlitas** [comment]
Yeah... Make a Ten is very broken right now. This is part of why we have been deferring a re-publish for region and culture. I'm unassigning and marking as blocking any upcoming publications for this sim.

**Comment by amanda-phet** [comment]
Is this a duplicate of https://github.com/phetsims/make-a-ten/issues/304? 

There are a number of reasons we need to re-publish this sim, and it's unfortunate it is still not prioritized.

**Comment by samreid** [comment]
@catherinecarter does this seem fixed to you?

**Comment by catherinecarter** [comment]
@samreid - Nope. I just went to bayes (unbuilt version), opened any of the game levels, and the same layout issue is present. 

<img width="160" height="86" alt="Image" src="https://github.com/user-attachments/assets/e8213c74-f3cb-418c-82d9-2a0b288a0a2e" />

Is there a different version I should be reviewing?

**Comment by catherinecarter** [comment]
@samreid was able to pull, upon which I refreshed and re-opened the sim. The game layout no longer looks broken, so this is great! 

Example:
<img width="339" height="72" alt="Image" src="https://github.com/user-attachments/assets/21e836bd-4c16-4bd3-a21c-2cd813b66ed7" />
